src/pages/RegistrationPage/RegistrationPage.module.css

```diff
@@ -0,0 +1,75 @@
+.register {
+  /* background-image: url('../../images/forms/road-1x.png'); */
+  background-repeat: no-repeat;
+  background-position: -218px 97px;
+  padding-bottom: 67px;
+  background-size: 666px 885px;
+}
+
+/* @media only screen and (-webkit-min-device-pixel-ratio: 2),
+only screen and (min-resolution: 192dpi) {
+  .register {
+    background-image: url('../../images/forms/road-2x.png');
+  }
+} */
+
+.registerWrapper {
+  display: flex;
+  flex-direction: column;
+  max-width: 610px;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.socialWrapper {
+  display: flex;
+  flex-direction: column;
+  margin-top: 32px;
+  gap: 32px;
+}
+
+.socialBtns {
+  display: flex;
+  gap: 16px;
+  justify-content: center;
+}
+
+.linkedinButton {
+  background-color: rgba(10, 102, 194, 1);
+}
+
+.linkedinButton:hover,
+.linkedinButton:focus,
+.linkedinButton:active {
+  border-color: rgb(13, 86, 160);
+  background-color: rgb(13, 86, 160);
+}
+
```

Comment:
we should decide how to write colors: hex or rgb (i don't mean rgb with alfa channel rgba())